### PR TITLE
Qgis

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: 10-gis.Rmd [hunk 1]
```diff
@@ -246,26 +246,38 @@ One of the algorithms on the above list, `"native:union"`, sounds promising.
 The next step is to find out what this algorithm does and how we can use it.
 This is the role of the `qgis_show_help()`, which returns a short summary of what the algorithm does, its arguments, and outputs.^[We can also extract some of information independently with `qgis_description()`, `qgis_arguments()`, and `qgis_outputs()`.]
 This makes it output rather long.
-Here, we only present the arguments`"native:union"` expects.
+The following command returns a data frame with each row representing an argument required by `"native:union"` and columns with the name, description, type, default value, available values, and acceptable values associated with each:
 
 ```{r 09-gis-6, eval=qgis}
-alg = "native:union"
-qgis_arguments(alg) |>
-  dplyr::mutate(acceptable_values = unlist(acceptable_values)) |>
```
Comment (Robinlovelace) [inline 10-gis.Rmd:219]: This line caused the following error: https://github.com/geocompx/geocompr/actions/runs/4647331273/jobs/8224278198#step:5:2000

**Comment by jannes-m** [comment]
Excellent work. And if there is no rush, I will review when we are back from our vacations.

**Comment by Robinlovelace** [comment]
Heads-up @jannes-m going to merge now as want to mention in response to review round II, that I'm about to send to CRC...

**Comment by Robinlovelace** [comment]
Post merge review v. welcome. Email incoming!

**Comment by Robinlovelace** [comment]
Heads-up @jannes-m the merge has definitely revealed some issues: https://r.geocompx.org/gis.html 

Pros and cons: at least they are visible there for all to see now... apologies for missing some issues in this hasty PR, there's a lot going on and keen to keep the main branch moving.

**Pull request merged**